pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -272,12 +303,41 @@ func (g *Gatherer) updateAlertsCacheFromClient(ctx context.Context, metricsClien
 	return nil
 }
 
+func (g *Gatherer) updateVersionCache(ctx context.Context, configClient configv1client.ConfigV1Interface) error {
+	clusterVersion, err := configClient.ClusterVersions().Get(ctx, "version", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	g.clusterVersion = clusterVersion.Spec.DesiredUpdate.Version
```

Comment:
I think this is not correct. Is it? `DesiredUpdate` attribute/field is optional. I think we want `clusterVersion.Status.Desired.Version`